numpy/lib/tests/test_arraypad.py

```diff
@@ -1260,3 +1260,38 @@ def test_check_wrong_pad_amount(self):
         kwargs = dict(mode='mean', stat_length=(3, ))
         assert_raises(TypeError, pad, arr, ((2, 3, 4), (3, 2)),
                       **kwargs)
+
+
+@pytest.mark.xfail(reason="not yet implemented, see gh-11358")
+def test_unsupported_mode():
+    """Test error message for unsupported pad modes."""
```

Comment:
Oh, one thing, unless this changed with pytest, please avoid docstrings and use comments instead. At least nose would replace test function with the doc string, which tends to be a bit annoying.

Reply:
pytest does not do this by default, so I think we should leave it as is.